### ISSUE TITLE
support m5/c5 class machines

### ISSUE THREAD

**Issue by jasonhancock** [graymeta/terraform-aws-platform#32]
There's an error related to the network interfaces when spinning up m5/c5 class machines.

**Comment by jasonsattler** [comment]
c5.large worked for the ECS Cluster.  

For m5.large I got the following
```
* module.platform.module.services.aws_cloudformation_stack.services_asg: 1 error(s) occurred:

* aws_cloudformation_stack.services_asg: ROLLBACK_COMPLETE: ["The following resource(s) failed to create: [AutoScalingGroup]. . Rollback requested by user." "Group did not stabilize. {current/minSize/maxSize} group size = {0/2/4}. Failed Scaling Activity: Enhanced networking with the Elastic Network Adapter (ENA) is required for the 'm5.large' instance type. Ensure that you are using an AMI that is enabled for ENA. Launching EC2 instance failed."]
* module.platform.module.services.aws_lb.services_alb: Resource 'aws_security_group.services_alb_security_group' not found for variable 'aws_security_group.services_alb_security_group.id'
* module.platform.module.services.aws_security_group.services_alb_security_group: "ingress.3.cidr_blocks.0" must contain a valid CIDR, got error parsing: invalid CIDR address: 34.214.91.138

Terraform does not automatically rollback in the face of errors.
Instead, your Terraform state file has been partially updated with
any resources that successfully completed. Please address the error
above and apply again to incrementally change your infrastructure.
```

**Comment by jasonsattler** [comment]
When looking at the initramfs I found that both ena and nvme was not in the image.  Tried to do a dracut/mkinitrd but it will show a bunch of warnings that unable to find path to driver file.  So I found these utilities work on what is defined in /lib/modules/$(uname -r)/modules.dep which before the aws import was full of relative paths to all the drivers.  After the import I found full path to the drivers.  To fix the issue I ran depmod which rebuild the modules.dep and also fixed the dracut/mkinitrd commands.  In the build process I added the ena and nvme drivers to /etc/dracut.conf file so they should be added to the initramfs image.

https://github.com/graymeta/mf2/pull/3367
